client/src/components/Navigation.jsx

```diff
@@ -0,0 +1,62 @@
+import { useState } from 'react'
+import { NavLink } from 'react-router-dom'
+import styled from 'styled-components/macro'
+import Hamburger from './Hamburger'
+
+export default function Navigation() {
+  const [active, setActive] = useState(null)
+  return (
+    <>
+      <NavContainer className={active ? 'active' : ''}>
+        <NavLink className="link-2" to="/tbr">
+          Books
+        </NavLink>
+        <NavLink className="link-3" to="/monthly-tbr">
+          Monthly TBR
+        </NavLink>
+        <NavLink className="link-4" to="/add">
+          Add book to TBR
+        </NavLink>
+        <NavLink className="link-1" role="button" to="/">
```

Comment:
Not sure whether the role is necessary. But if you add it to one link, you might as well add it to the others as well :)